docker-compose-external-pg.yml

```diff
@@ -34,16 +34,7 @@ services:
       -c shared_preload_libraries=pg_stat_statements
       -c pg_stat_statements.track=all 
       -c max_connections=200
+#      -c psql "postgresql://postgres:pmm_password@localhost/grafana" -c 'CREATE EXTENSION IF NOT EXISTS pg_stat_statements;'
```

Comment:
What's the reason for this to be commented out?

Reply:
removed